Packs/QualysVulnerabilityManagement/Integrations/QualysVulnerabilityManagement/QualysVulnerabilityManagement.py

```diff
@@ -0,0 +1,931 @@
+    Raises:
+        KeyError: will be raised by parse_two_keys_dict if response has unexpected keys
+        TypeError: will be raised by  parse_two_keys_dict response is not a dictionary
+    """
+    command_data = kwargs['command_data']
+    response = kwargs['json_response']
+    single_val_dict = parse_two_keys_dict(response)
+    readable_output = tableToMarkdown(name=command_data['table_name'], t=single_val_dict)
+    return single_val_dict, readable_output
+
+
+def build_single_text_output(**kwargs) -> Tuple[Dict[str, Any], str]:
+    """
+    creates output with the dictionary returned from the request and the text attached to it
+    Args:
+        **kwargs:
+                json_response (Dict): response received from Qualys API
+    Returns:
+            Tuple containing a dictionary and the text returned in the response
+    """
+    output = kwargs['json_response']
+    readable_output = output['TEXT']
+    return output, readable_output
+
+
+def build_unparsed_output(**kwargs) -> Tuple[Dict[str, Any], str]:
+    """
+    creates output with the dictionary returned from the request and a markdown table generated
+    from the unparsed response received
+    Args:
+        **kwargs:
+                json_response (Dict): response received from Qualys API
+                command_data (Dict): data about a specific command from the general commands_data dict
+    Returns:
+            Tuple containing a dictionary and a markdown table generated from the response
+    """
+    command_data = kwargs['command_data']
+    unparsed_output = kwargs['json_response']
+    readable_output = tableToMarkdown(name=command_data['table_name'], t=unparsed_output)
+    return unparsed_output, readable_output
+
+
+def build_ip_list_output(**kwargs) -> Tuple[Dict[str, List[str]], str]:
+    """
+    creates output with a new dictionary parsed from the original response and two markdown tables generated
+    for the ip-list command
+    Args:
+        **kwargs:
+                json_response (Dict): response received from Qualys API
+                command_data (Dict): data about a specific command from the general commands_data dict
+    Returns:
+            Tuple containing a dictionary created and a markdown table generated from the response
+    Raises:
+            KeyError: can be raised by either change_dict_keys or create_ip_list_dicts
+            TypeError: can be raised by either change_dict_keys or create_ip_list_dicts
+    """
+    command_data = kwargs['command_data']
+    json_response = kwargs['json_response']
+    asset_collection = json_response[command_data['collection_name']]
+
+    parsed_output = change_dict_keys(command_data['new_names_dict'], asset_collection)
+    list_of_fields = create_ip_list_dicts(parsed_output)
+    readable_output = ""
+    for field in list_of_fields:
+        readable_output += tableToMarkdown(name=command_data['table_name'], t=field)
```

Comment:
Yes it does happen, I made two headers because there are two tables. Although I can change it and just put one header for both of them. I will make this change.